includes/classes/Command.php

```diff
@@ -593,11 +593,7 @@ public function index( $args, $assoc_args ) {
 		 */
 		do_action( 'ep_wp_cli_pre_index', $args, $assoc_args );
 
-		if ( defined( 'EP_IS_NETWORK' ) && EP_IS_NETWORK ) {
-			set_site_transient( 'ep_wpcli_sync', true, $this->transient_expiration );
-		} else {
-			set_transient( 'ep_wpcli_sync', true, $this->transient_expiration );
-		}
+		set_transient( 'ep_wpcli_sync', true, $this->transient_expiration );
```

Comment:
Let's annotate here that this is a "hack" and differs from upstream on purpose, so that the next time we merge this won't get overwritten again.